### PR TITLE
added timestamp to the processor information to filter out outdated information

### PR DESCRIPTION
**Pull request by MGathier**

### CODE REVIEW

File: axonserver/src/main/java/io/axoniq/axonserver/component/processor/listener/ProcessorsInfoTarget.java [hunk 1]
```diff
@@ -64,20 +68,29 @@ public EventProcessorStatusUpdated onEventProcessorStatusChange(EventProcessorSt
...
     @EventListener
     @Order(0)
     public void onClientDisconnected(TopologyEvents.ApplicationDisconnected event) {
         clients.remove(event.getClientStreamId());
         cache.remove(event.getClientStreamId());
+        cleanupCache();
+    }
+
+    private void cleanupCache() {
+        long expired = System.currentTimeMillis() - expireTime;
```
Comment (m1l4n54v1c) [inline axonserver/src/main/java/io/axoniq/axonserver/component/processor/listener/ProcessorsInfoTarget.java:83]: Could we apstract time somehow to make testing easier?

File: axonserver/src/main/java/io/axoniq/axonserver/component/processor/ClientEventProcessorInfo.java [hunk 2]
```diff
@@ -29,6 +30,7 @@ public ClientEventProcessorInfo(String clientId, String clientStreamId, String c
         this.clientStreamId = clientStreamId;
         this.context = context;
         this.eventProcessorInfo = eventProcessorInfo;
+        this.timestamp = System.currentTimeMillis();
```
Comment (m1l4n54v1c) [inline axonserver/src/main/java/io/axoniq/axonserver/component/processor/ClientEventProcessorInfo.java:33]: It'd be nice that this class remains the simple data holder :)

**Review by m1l4n54v1c — COMMENTED** [review]
Testability improvements.

File: axonserver/src/main/java/io/axoniq/axonserver/component/processor/ClientEventProcessorInfo.java (hunk 2 above)
Comment (MGathier) [reply axonserver/src/main/java/io/axoniq/axonserver/component/processor/ClientEventProcessorInfo.java]: removed timestamp from here.

**Review by saratry — APPROVED** [review]
LGTM!

**Pull request merged**